reservations/views.py

```diff
@@ -53,4 +53,27 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"MESSAGE": "KEY_ERROR"}, status=400)
         except Room.DoesNotExist:
-            return JsonResponse({"MESSAGE": "DOESNOT_EXIST_ROOM"}, status=400)
\ No newline at end of file
+            return JsonResponse({"MESSAGE": "DOESNOT_EXIST_ROOM"}, status=400)
+
+class DetailReservationView(View):
+    @signin_decorator
+    def get(self, request, reservation_number):
+        reservation          = Reservation.objects.select_related("room") \
+                                                  .prefetch_related("room__image_set") \
+                                                  .filter(user = request.user).get(number = reservation_number)
```

Comment:
```suggestion
                                                  .get(user = request.user, number = reservation_number)
```
굳이 filter에 get을 섞어 쓸 필요가 없어보여요!

Reply:
수정 후 pr요청 드리겠습니다.